### DISCUSSION TITLE
Multiple arguments in utils functions

### DISCUSSION THREAD

**Discussion by nob3110** [stitchesjs/stitches#1070] [Help] [answered]
It would make sense if utils could use multiple arguments.

For example, there is a way to parse a single string argument.
However, I don't think that method is appropriate for typescript.

Is there any other ideas to achieve like this?

```ts
createStitches({
  utils: {
    paddingVH: (v, h) => ({
      paddingTop: v,
      paddingBottom: v,
      paddingLeft: h,
      paddingRight: h,
    }),
  },
});
```

**Comment by sandgraham** [comment, marked as answer]
I've been using an array or object to wrap the arguments into a single argument.
```tsx
createStitches({
  utils: {
    paddingVH: ([v, h]: [number, number]) => ({
      paddingTop: v,
      paddingBottom: v,
      paddingLeft: h,
      paddingRight: h,
    }),
  },
});

css({ paddingVH: [2, 3] });
```
Can also use `Stitches.PropertyValue` to keep the types theme aware:
```tsx
type Size = Stitches.PropertyValue<'sizes'>;
createStitches({
  utils: {
    paddingVH: ([v, h]: [Size, Size]) => ({
      paddingTop: v,
      paddingBottom: v,
      paddingLeft: h,
      paddingRight: h,
    }),
  },
});

css({ paddingVH: ["$2", "$3"] });
```

**Reply by nob3110** [reply]
I seriously do not know why I did not come up with such a simple idea.

Thanks a lot.
Really your idea makes sense.
